app/services/form1010_ezr_attachments/file_type_validator.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'form1010_ezr/service'
+
+module Form1010EzrAttachments
+  class FileTypeValidator
+    def initialize(file)
+      @file = file
+    end
+
+    # This method was created because there's an issue on the frontend where a user can manually 'change' a
+    # file's extension via its name in order to circumvent frontend validation. '.xlsx' files, in particular, were
+    # the cause of several form submission failures. With that said, we need to check the actual file type
+    # and ensure the Enrollment System accepts it.
+    def validate_file_type
```

Comment:
Looks good! I like how it turned out. Only one small thing. Would this method be better named just `validate`? So it's  `Form1010EzrAttachments::FileTypeValidator.new(my_file).validate`? Not a big deal either way.

Reply:
oh, sure! That makes sense to me! I'll change it right quick.